spec/test_credentials.rb

```diff
@@ -0,0 +1,16 @@
+#
+# Fill in your gateway details below to test remotely (when not using VCR
+# prerecorded responses). This should only be neccessary when rerecording
+# existing actions or adding new remote tests
+#
+#
+# If you want git to ignore changes to this file (so you do not accidently
+# commit your API details then simply run:
+# git update-index --assume-unchanged spec/test_credentials.rb
+#
+
+# UNCOMMENT THE FOLLOWING CODE  if you wish to target UAT
+GATEWAY_ENDPOINT = 'https://bpoint-uat.premier.com.au/evolve/service.asmx'
+GATEWAY_LOGIN           = 'airtasker'
+GATEWAY_PASSWORD        = 'Pm0o4F4Xf0'
```

Comment:
Security note.. these were just test keys and had a shelf life of a few weeks at best. 
So best case scenario someone would have been able to run tests in our sandbox.  I believe I reset the keys right after the commit anyway.